src/constants.js

```diff
@@ -55,3 +55,6 @@ export const SAMPLER_TYPE_REMOTE = 'remote';
 // The value of the header is recorded as the tag on the root span, so that the
 // trace can be found in the UI using this value as a correlation ID.
 export const JAEGER_DEBUG_HEADER = 'jaeger-debug-id';
+
+// TRACER_BAGGAGE_HEADER_PREFIX is the default prefix used for saving baggage to a carrier.
+export const TRACER_BAGGAGE_HEADER_PREFIX = 'uberctx-';
```

Comment:
to be exact, this is "Jaeger specific", i.e. these constants must be the same across all jaeger tracers. We made these configurable in some implementations, but overall I don't think anyone deploying Jaeger outside of Uber would bother re-configuring the client libraries.